fdbserver/ConfigBroadcaster.actor.cpp

```diff
@@ -361,6 +296,8 @@ class ConfigBroadcasterImpl {
 		}
 	}
 
+	Future<Void> getError() const { return consumerFuture; }
```

Comment:
```suggestion
	Future<Void> getError() const { return consumerFuture || actors.getResult(); }
```

Reply:
This is still giving me trouble, I think because the `pushChanges` future is added to `actors`. Since that returns after a reply is received, that would then cause `getError` to return an error here wouldn't it?

Reply:
A single actor in the `ActorCollection` returning shouldn't cause any issue here. For the `ActorCollection` documentation:
```
//   - Throws an error immediately if any future throws an error
//   - Never returns otherwise, unless returnWhenEmptied=true in which case returns the first time it goes from count 1
//   to count 0 futures
```